src/components/WebexMeetingControl/WebexMeetingControl.jsx

```diff
@@ -7,6 +7,7 @@ import {useMeetingControl} from '../hooks';
 import Button from '../generic/Button/Button';
 import Icon from '../generic/Icon/Icon';
 import Select from '../generic/Select/Select';
+import {ROSTER_CONTROL, SHARE_CONTROL} from '../../adapters/MeetingsJSONAdapter';
```

Comment:
You should not import from adapters into the component code. How would work for other adapters? e.g. SDK adapter?